src/Form/WmContentContainerForm.php

```diff
@@ -133,18 +124,9 @@ public function form(array $form, FormStateInterface $form_state)
             '#description' => t('Allowed bundles in this type.'),
         ];
 
-        $child_bundles = [];
-        if ($entity->getHostEntityType()) {
-            $child_bundles = $this->getAllBundles($entity->getChildEntityType());
-        } elseif (isset($values['child_entity_type'])) {
```

Comment:
@brownrl Is it ok that we don't look in $values here anymore? Why were we doing this?